src/components/movie-info/movie-info.tsx

```diff
@@ -1,28 +1,32 @@
-import Button from '../button/button';
+import { useNavigate } from 'react-router-dom';
 import { Movie } from '../../types/movie';
+import Logo from '../logo/logo';
 import './style.css';
 
 type MovieInfoProps = {
-  onMovieInfoClose: () => void;
   movie: Movie | null;
 };
 
-export default function MovieInfo({ onMovieInfoClose, movie }: MovieInfoProps) {
+export default function MovieInfo({ movie }: MovieInfoProps) {
+  const navigate = useNavigate();
+
   const duration =
     movie &&
     `${Math.floor(movie.runtime / 60)}h 
```

Comment:
Let's move this logic to a separate helper/utility function.

Reply:
Moved to utilities file.